gpii/node_modules/flowManager/test/UntrustedSettingsDataSourceTests.js

```diff
@@ -80,23 +98,30 @@ gpii.tests.untrustedSettingsDataSource.setUpNock = function (config) {
     // mock POST requests to "/access_token"
     cloudMock.post(config.accessToken.url, config.accessToken.data)
         .reply(config.accessToken.status, config.accessToken.response);
+
     // mock GET requests to "/untrusted-settings"
-    cloudMock.get(config.untrustedSettings.url)
-        .matchHeader(config.untrustedSettings.authHeader.header, config.untrustedSettings.authHeader.value)
-        .reply(config.untrustedSettings.status, config.untrustedSettings.response);
+    cloudMock.get(config.untrustedSettingsGet.url)
+        .matchHeader(config.untrustedSettingsGet.authHeader.header, config.untrustedSettingsGet.authHeader.value)
+        .reply(config.untrustedSettingsGet.status, config.untrustedSettingsGet.response);
+
+    // mock PUT requests to "/untrusted-settings"
+    cloudMock.put(config.untrustedSettingsPut.url)
```

Comment:
Are we able to put a matcher here that verifies the preferences being sent? At the moment these tests do not verify that the preferences are being sent along to the cloud. If I modify `gpii/node_modules/flowManager/src/UntrustedSettingsDataSource.js` so that an empty object is sent instead of the specified preferences, the tests still pass.